src/Evolution/Executables/ScalarWave/EvolveScalarWave.hpp

```diff
@@ -138,11 +139,10 @@ struct EvolutionMetavars {
                                   volume_dim, Tags::Time, observe_fields,
                                   analytic_solution_fields>,
                               Events::time_events<system>>>>,
-        tmpl::pair<
-            StepChooser<StepChooserUse::LtsStep>,
-            tmpl::push_back<
-                StepChoosers::standard_step_choosers<system>,
-                StepChoosers::ByBlock<StepChooserUse::LtsStep, volume_dim>>>,
+        tmpl::pair<StepChooser<StepChooserUse::LtsStep>,
+                   tmpl::push_back<StepChoosers::standard_step_choosers<system>,
+                                   StepChoosers::ByBlock<
+                                       StepChooserUse::LtsStep, volume_dim>>>,
```

Comment:
Did anything meaningful change in this file?  It looks like just formatting and an unused include.

The commit message might also need an update.

Reply:
Agreed, I'll drop these changes -- originally I added the `ErrorControl` chooser only to scalar wave, but figured it should be fine to just add it globally via`Factory.hpp` instead.